Gems/Atom/RPI/Code/Source/RPI.Public/Model/ModelLod.cpp

```diff
@@ -345,95 +345,6 @@ namespace AZ
             return success;
         }
 
-        bool ModelLod::GetStreamsForMesh(
```

Comment:
Why is this method removed?

Reply:
This was a single device copy of the method that we temporarily added while we were transitioning only parts of the code base to multi device. The original (now multi device) version of the method is still there.